bundles/binding/org.openhab.binding.gc100ir/src/main/java/org/openhab/binding/gc100ir/internal/GC100IRConnector.java

```diff
@@ -0,0 +1,183 @@
+/**
+ * Copyright (c) 2010-2016, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.gc100ir.internal;
+
+import java.io.IOException;
+import java.net.UnknownHostException;
+import java.util.HashMap;
+import java.util.concurrent.ExecutionException;
+
+import org.openhab.binding.gc100ir.internal.response.GC100IRCommand;
+import org.openhab.binding.gc100ir.internal.response.GC100IRCommandCode;
+import org.openhab.binding.gc100ir.lib.GC100IRControlPoint;
+import org.openhab.binding.gc100ir.util.GC100ItemBean;
+import org.openhab.binding.gc100ir.util.GC100ItemUtility;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Manages the TCP/IP socket connection for a single GC100 instance.
+ *
+ * @author Parikshit Thakur & Team
+ * @since 1.9.0
+ */
+public class GC100IRConnector {
+
+    private static final Logger logger = LoggerFactory.getLogger(GC100IRConnector.class);
+
+    // the GC100 instance
+    private final GC100IRHost gc100;
+    private GC100IRControlPoint gc100ControlPoint;
+
+    private boolean connected = false;
+
+    private static GC100ItemUtility GC100ItemUtility;
```

Comment:
Could you change this to 
```java
    private static GC100ItemUtility gc100ItemUtility;
```
or similar?  It's odd that a static class variable can have the same name as a class!